package.json

```diff
@@ -74,14 +74,12 @@
   },
   "scripts": {
     "dev": "next dev",
-    "build:icons": "svgstore public/icons/* | svgo -i - -o public/icons.svg --disable=removeUselessDefs,cleanupIDs",
-    "prebuild": "run-p build:icons",
```

Comment:
Still, removing this will slow down development of new features and make the releases harder. I would keep the process the same: move everything possible but continue development in the project. The next time move everything again.

Reply:
As you can see in the commit message I don't say we shouldn't use it. We can add it back when needed but for now, we don't use it's not possible to comment these two lines in json file.